pyqir-parser/Cargo.toml

```diff
@@ -9,21 +9,16 @@ readme = "README.md"
 homepage = "https://github.com/qir-alliance/pyqir"
 repository = "https://github.com/qir-alliance/pyqir"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-llvm-ir = { version = "0.8.1" }
 inkwell = { git = "https://github.com/TheDan64/inkwell", branch = "master", default-features = false, features = ["target-x86"] }
-pyo3 = { version="0.15.2", optional = true }
+llvm-ir = { version = "0.8.1" }
+pyo3 = { version = "0.17.2", features = ["abi3-py37", "extension-module"] }
 
 [features]
-extension-module = ["pyo3/abi3-py36", "pyo3/extension-module"]
-python-bindings = []
 llvm11-0 = ["llvm-ir/llvm-11", "inkwell/llvm11-0"]
 llvm12-0 = ["llvm-ir/llvm-12", "inkwell/llvm12-0"]
 llvm13-0 = ["llvm-ir/llvm-13", "inkwell/llvm13-0"]
-#llvm14-0 = ["llvm-ir/llvm-14", "inkwell/llvm14-0"]
-default = ["extension-module", "python-bindings"]
+llvm14-0 = ["llvm-ir/llvm-14", "inkwell/llvm14-0"]
 
```

Comment:
@samarsha Why is the `python-bindings`  feature removed?

Reply:
I added it back for pyqir-parser.